src/main/java/org/gaul/s3proxy/S3ProxyHandler.java

```diff
@@ -1654,7 +1667,8 @@ private void handleListParts(HttpServletRequest request,
             xml.writeDefaultNamespace(AWS_XMLNS);
 
             writeSimpleElement(xml, "Bucket", containerName);
-            writeSimpleElement(xml, "Key", blobName);
+            writeSimpleElement(xml, "Key", encodeBlob(
+                    request.getParameter("encoding-type"), blobName));
```

Comment:
According to that link, the ancestor is ListBucketResult not ListPartsResult. Not sure if that's a typo, but that isn't in the ListPartsResult area. If you want I can add it and assume it's a typo in the documentation. See Ancestor in attached screenshot. (The ancestor of all the other items is ListPartsResult)

<img width="907" alt="screen shot 2015-12-14 at 4 48 28 pm" src="https://cloud.githubusercontent.com/assets/884727/11795055/8edba5d4-a282-11e5-9268-984b0f4487dc.png">